pkg/plugin/plugin_test.go

```diff
@@ -447,6 +448,53 @@ var _ = Describe("CNI Plugin", func() {
 				testSplitVlanIds(trunks, nil, errors.New("incorrect trunk maxID parameter"), false)
 			})
 		})
+		Context("purge ports with failed interfaces", func() {
+			const IFNAME = "eth0"
+
+			conf := fmt.Sprintf(`{
+				"cniVersion": "0.3.1",
+				"name": "mynet",
+				"type": "ovs",
+				"OvnPort": "test-port",
+				"bridge": "%s"}`, BRIDGE_NAME)
+
+			It("DEL removes ports without network namespace", func() {
+				firstTargetNs, err := testutils.NewNS()
+				Expect(err).NotTo(HaveOccurred())
+				defer firstTargetNs.Close()
+
+				secondTargetNs, err := testutils.NewNS()
+				Expect(err).NotTo(HaveOccurred())
+				defer secondTargetNs.Close()
+
+				firstResult := attach(firstTargetNs, conf, IFNAME, "", "test-port-1")
+				secondResult := attach(secondTargetNs, conf, IFNAME, "", "test-port-2")
+
+				firstHostIface := firstResult.Interfaces[0]
```

Comment:
Please add a comment describing that you are removing the interface used by the port to make it faulty for OVS